Week1/homework/js-exercises/logRemainder.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+let x = 7;
+let y = 21;
+let z = 13;
+console.log(x + " modulo 3 is equal to " + (x%3)); 
+// x%3 gives us the remainder when x divided by 3
+console.log(y + " modulo 4 is equal to " + (y%4));
+ // y%4 gives us the remainder when y divided by 4
+console.log(z + " modulo 2 is equal to " + (z%2)); 
+// z%2 gives us the remainder when z divided by 2
```

Comment:
👍 for logic but can use `const` here since you aren't reassigning `x`, `y` or `z`